django/test/utils.py

```diff
@@ -113,21 +113,25 @@ def setup_test_environment(debug=None):
     if debug is None:
         debug = settings.DEBUG
 
-    _SavedSettings.debug = settings.DEBUG
-    settings.DEBUG = debug
+    # Storing previous values in the settings module itself is problematic.
```

Comment:
I think this comment is obsolete. With the changes, I wouldn't think of refactoring it to store things in the settings module instead.